core/crypto.mk

```diff
@@ -37,7 +37,14 @@ CFG_CRYPTO_SM3 ?= y
 CFG_CRYPTO_DSA ?= y
 CFG_CRYPTO_RSA ?= y
 CFG_CRYPTO_DH ?= y
+# ECC includes ECDSA and ECDH
 CFG_CRYPTO_ECC ?= y
+ifneq ($(CFG_CRYPTOLIB_NAME),mbedtls)
```

Comment:
I'd prefer `ifeq ($(CFG_CRYPTOLIB_NAME),tomcrypt)` since that's the only place where it's currently implemented.

Reply:
Agreed, makes more sense.

Reply:
Yes, will fix.